CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ A proper header must be in place for any file contributed to Eclipse Kura. For a
 
 ```
 /*******************************************************************************
- * Copyright (c) 2016 <legal entity> and others
+ * Copyright (c) 2017 <legal entity> and others
```

Comment:
I donno why it didn't appear before but when I used it as code in the comment it appeared.